setup.py

```diff
@@ -647,9 +647,7 @@ def build_extensions(self):
         ]
         # to avoid a meaningless warning
         self._pyfftw_define_macros.append(("NPY_NO_DEPRECATED_API", "NPY_1_7_API_VERSION"))
-        if get_platform() in ('win32', 'win-amd64'):
-            # because Numpy 2.0 includes complex.h
-            self._pyfftw_define_macros.append(("FFTW_NO_Complex", "1"))
+        self._pyfftw_define_macros.append(("FFTW_NO_Complex", 1))
```

Comment:
I guess there are some cases for which it makes sense to include `complex.h`. I remember that it changes the behavior of fftw, see https://www.fftw.org/fftw3_doc/Complex-numbers.html.

Reply:
In fftw3.h, there is:

```
/* If <complex.h> is included, use the C99 complex type.  Otherwise
   define a type bit-compatible with C99 complex */
#if !defined(FFTW_NO_Complex) && defined(_Complex_I) && defined(complex) && defined(I)
#  define FFTW_DEFINE_COMPLEX(R, C) typedef R _Complex C
#else
#  define FFTW_DEFINE_COMPLEX(R, C) typedef R C[2]
#endif
```

Reply:
pyFFTW defines the complex types as arrays, which are not compatible with `_Complex double` etc (#396).

https://github.com/pyFFTW/pyFFTW/blob/89e2d2c208b7be6298625ab19edb9f9cc0d31303/pyfftw/pyfftw.pxd#L80-L82